README.md

```diff
@@ -53,6 +53,8 @@ The **org.freedesktop.ratbag1.Profile** interface provides:
 - Methods:
   - SetActive(void) -> set this profile to be the active profile
   - GetResolutionByIndex(uint) -> returns the object path for the given index
+- Signals:
+  - ActiveProfileChanged -> active profile changed, carries index of the currently active profile
```

Comment:
We might want to change the wording here. "Currently active profile" may also be interpreted as the profile that's active when the active profile change occurs. Thoughts?

Same applies for the signal description of the active resolution.